shared/remediations/bash/file_ownership_var_log_audit.sh

```diff
@@ -0,0 +1,15 @@
+# platform = multi_platform_rhel
+
+if `grep -q ^log_group /etc/audit/auditd.conf` ; then
+  GROUP=$(awk -F "=" '/log_group/ {print $2}' /etc/audit/auditd.conf | tr -d ' ')
+  if ! [ "${GROUP}" == 'root' ] ; then
+    chown root.${GROUP} /var/log/audit
+    chown root.${GROUP} /var/log/audit/audit.log*
```

Comment:
These changes look good to me. But should we also update / enhance existing OVAL check for ```file_ownership_var_log_audit.xml``` file?

Suppose ```log_group != root``` will be configured in ```/etc/audit/auditd.conf``` - the above code snippet will then set up ownerhisp of ```/var/log/audit``` directory and ```/var/log/audit/audit.log*``` files to that group.

But subsequent run  of OVAL scan for this property will return 'error' (OVAL check for ```file_ownership_var_log_audit``` will return 'fail' because the only user account ```/var/log/audit``` and ```/var/log/audit/*``` files are currently allowed to be under is user==0 and group==0 per:

&nbsp; &nbsp; [1] https://github.com/OpenSCAP/scap-security-guide/blob/master/shared/oval/file_ownership_var_log_audit.xml

So IMHO if we want the remediation script to possibly set /var/log/audit | /var/log/audit/audit.log* to some other user.group ownership than root.root, we should modify also the OVAL to allow it (otherwise the remediation will still return failure when this case happens - user account other than 'root' will be set in ```log_group``` key of ```auditd.conf```).

What do you think?